nativescript-angular/tslint.json

```diff
@@ -57,7 +57,7 @@
     "no-empty": false,
     "no-eval": true,
     "no-null-keyword": false,
-    "no-shadowed-variable": true,
+    "no-shadowed-variable": false,
```

Comment:
Is there a reason to disable this rule?
Tslint was OK locally when I enabled it back again